### PR TITLE
adjust cwe_checker plugin to new cwe_checker version

### PR DESCRIPTION
**Pull request by Enkelmann**
This PR updates the cwe_checker plugin to be compatible with the newest cwe_checker docker image interface.

### CODE REVIEW

**Comment by Enkelmann** [comment]
Argh, the CI is again failing because of `lief`. As soon as #531 is merged, I will merge the master into this branch to get some useful CI-output.

**Review by jstucke — APPROVED** [review]
Looks fine. Can be merged when the CI builds are successful.

**Pull request merged**
